backend/app/modules/dto.py

```diff
@@ -80,6 +80,14 @@ class UserAllocationRequestPayload(JSONWizard):
     signature: str
 
 
+@dataclass(frozen=True)
+class ScoreDelegationPayload(JSONWizard):
+    primary_addr: str
+    secondary_addr: str
+    primary_addr_signature: Optional[str] = None
+    secondary_addr_signature: Optional[str] = None
```

Comment:
`signatures: Optional[Tuple[str,str]]` ?

Avoids situation when one is set and other is not on type level.

Reply:
If there is only one we'll reject the request. I'm leaving it as is, because it's bigger refactor to do it